### ISSUE TITLE
Setting owner of plot

### ISSUE THREAD

**Issue by Xayanix** [IntellectualSites/PlotSquared#1686]
Players are able to change owner of their plot without confirmation from new owner.

**Comment by braceyourself07** [comment]
Are your players op'd or given out the * or plots.* perm?   Will want to reference https://github.com/IntellectualSites/PlotSquared/wiki/Commands-and-Permissions#base-permissions in setting up your permissions.  Because it sounds like somehow your players are getting the plots.admin.command.setowner permission.

**Comment by Xayanix** [comment]
I mean that every player (named for example: player1) can make trash plots and give it to another player (player2), then player2 has a lot of plots which he dont want to have.

**Comment by braceyourself07** [comment]
What command are they using to give it to another player?  /plot setowner?  If so you could set the negated perm -plots.set.owner to disable that command.

**Comment by Xayanix** [comment]
Yeah, but when i will block /p setowner then people wont be able to give legit plots. PlotSquared just need an additional command to accept or decline receiving plot from another player.

**Comment by braceyourself07** [comment]
Oh I see what you're saying.

**Comment by dordsor21** [comment]
Feature request for a /ps give or something, as setowner is an admin command
